kube-state-metrics.yaml

```diff
@@ -30,12 +22,28 @@ pipeline:
         github.com/emicklei/go-restful/v3@v3.11.3
       modroot: .
 
-  - runs: |
-      mkdir -p ${{targets.destdir}}/usr/bin
-      make build-local
-      cp kube-state-metrics ${{targets.destdir}}/usr/bin
+  - uses: go/build
+    with:
+      packages: .
+      output: kube-state-metrics
+      ldflags: |
+        -X github.com/prometheus/common/version.Version=${{package.version}}
+        -X github.com/prometheus/common/version.Revision=$(git rev-parse --short HEAD)
+        -X github.com/prometheus/common/version.Branch=$(git rev-parse --abbrev-ref HEAD)
+        -X github.com/prometheus/common/version.BuildUser=wolfi
+        -X github.com/prometheus/common/version.BuildDate=$(date -u +'%Y-%m-%dT%H:%M:%SZ')
 
-  - uses: strip
+subpackages:
+  - name: ${{package.name}}-bitnami-compat
+    description: "compat package for ${{package.name}} image"
+    pipeline:
+      - uses: bitnami/compat
+        with:
+          image: ${{package.name}}
+          version-path: 2/debian-12
+      - runs: |
+          chmod g+rwX ${{targets.contextdir}}/opt/bitnami
+          find / -perm /6000 -type f -exec chmod a-s {} \; || true
```

Comment:
Wouldn't this also try to modify the entire build environment? Should find be done inside `${{targets.contextdir}}/opt/bitnami` instead?

Reply:
copy-paste from other configs, I think we don't need it. let me remove it or use `find ${{targets.contextdir}}/ -perm /6000 -type f -exec chmod a-s {} \; || true`